system-int-test/test/cmr/system_int_test/search/scrolling_search_test.clj

```diff
@@ -79,6 +80,26 @@
                                          {:headers {routes/SCROLL_ID_HEADER scroll-id}})]
             (is (= (count all-grans) hits))
             (is (data2-core/refs-match? [] result))))))
+    
+    (testing "Scrolling with different search params from the original"
+      (let [result (search/find-concepts-in-format
+                     mime-types/xml
+                     :granule
+                     {:provider "PROV1" :scroll true :page-size 2 :result-format mime-types/xml})
+            format (get-in result [:headers :Content-Type])
+            hits (get-in result [:headers :CMR-Hits])
+            scroll-id (get-in result [:headers :CMR-Scroll-Id])
+            ;; Do a subsequent scroll search with different format, provider, scroll and page-size.
+            ;; Verify the new search params are ignored. It still returns the same xml format.
+            subsequent-result (search/find-concepts-in-format
+                                mime-types/json
+                                :granule
+                                {:provider "PROV2" :scroll false :page-size 10}
+                                {:headers {routes/SCROLL_ID_HEADER scroll-id}})
+            subsequent-format (get-in subsequent-result [:headers :Content-Type])
+            subsequent-hits (get-in subsequent-result [:headers :CMR-Hits])]
+         (is (= hits subsequent-hits))
```

Comment:
Can you add in one more verification that the response body contains 2 items and not 10? Also for the test setup do we have more than 4 granules that would have matched the original query?

Reply:
The previous existing test already covers this: 
(testing "Subsequent searches gets original page-size results"
          (let [result (search/find-refs :granule
                                         {:scroll true :page-size 10}
                                         {:headers {routes/SCROLL_ID_HEADER scroll-id}})]
            (is (= (count all-grans) hits))
            (is (data2-core/refs-match? [gran3 gran4] result))))

Reply:
And yes. we have 5 granules that match the original query.